cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/analyzer/RandomSelfHealingTest.java

```diff
@@ -83,38 +85,55 @@
     singleDeadBroker.put(ClusterProperty.NUM_DEAD_BROKERS, 1);
     for (Map.Entry<Integer, String> entry : goalNameByPriority.entrySet()) {
       Object[] singleDeadSingleSoftParams = {singleDeadBroker, Collections.singletonMap(entry.getKey(),
-          entry.getValue()), balancingConstraint};
+          entry.getValue()), balancingConstraint, Collections.emptySet()};
       params.add(singleDeadSingleSoftParams);
+      Object[] singleDeadSingleSoftParamsWithExcludedTopics = 
+          {singleDeadBroker, Collections.singletonMap(entry.getKey(), entry.getValue()), balancingConstraint, 
+          Collections.singleton("T0")};
+      params.add(singleDeadSingleSoftParamsWithExcludedTopics);
     }
-    props.setProperty(KafkaCruiseControlConfig.MAX_REPLICAS_PER_BROKER_CONFIG, Long.toString(5000L));
+    props.setProperty(KafkaCruiseControlConfig.MAX_REPLICAS_PER_BROKER_CONFIG, Long.toString(5100L));
```

Comment:
Was 5K causing test failures?

Reply:
I have seen the ReplicaCapacityGoal been failing from time to time even without this patch. We need to dig a bit further to see why that is the case. This change did not solve the problem so we don't really need it.